components/dashboard/src/AppNotifications.tsx

```diff
@@ -62,7 +62,7 @@ const UPDATED_PRIVACY_POLICY = (updateUser: (user: Partial<UserProtocol>) => Pro
 export function AppNotifications() {
     const [topNotification, setTopNotification] = useState<Notification | undefined>(undefined);
     const { user, loading } = useUserLoader();
-    const updateUser = useUpdateCurrentUserMutation();
+    const { mutateAsync } = useUpdateCurrentUserMutation();
```

Comment:
Needed to destructure this as it was causing infinite re-renders since useMutation returns a new instance every time. We should probably do a sweep to make sure we're not using it directly as a dependency in any `useEffect()` calls.

Reply:
How did you notice the infinite re-renders? A drop in perf? I'm curious

Reply:
Yah, typing in an input got really sluggish, then I checked the console and noticed a flood of react errors (can't remember exact error, wasn't that helpful other than made me realize something was causing re-renders over and over).